src/Components/SysTable/SysTable.js

```diff
@@ -89,11 +100,10 @@ const SysTable = () => {
             }}
             noSystemsTable={<MessageState className='pf-c-card' icon={SearchIcon} variant='large' title={intl.formatMessage(messages.noResults)}
                 text={intl.formatMessage(messages.noResultsMatch)}/>}
-            customFilters={{
```

Comment:
I believe that you can pass the value of match filter into custom params and inventory itself fires onRefreshData as soon as the filter value changes. This also enables not to have useEffect to fire onRefreshData from malware app (line 38)